src/libmongoc/tests/test-conveniences.c

```diff
@@ -1873,25 +1904,24 @@ semver_parse (const char *str, semver_t *out)
       return;
    }
    out->has_minor = true;
-   out->minor = (int) bson_ascii_strtoll (str, &dot, 10);
+   out->minor = (int) bson_ascii_strtoll (dot, &dot, 10);
```

Comment:
This was a bug fix, `str` always points to the start of the string, `dot` points to the next section of the string to read.